string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,21 @@
+/* exported capitalizeWord */
+// input: word
+// output: word with first char uppercase, rest lowercase, but if word is javascript, always cased "JavaScript"
+// first character make uppercase and assign to storage for output
+// look at each character
+// assess if word is javascript
+// if true, return word "JavaScript"
+// otherwise, check each characters one by one and add to storage for output in all lowercase
+// return output
+
+function capitalizeWord(word) {
+  var str = word[0].toUpperCase();
+  for (var i = 1; i < word.length; i++) {
+    if (word.toLowerCase() === 'javascript') {
+      return 'JavaScript';
```

Comment:
I would move this to be the first thing in the function. If the word is JavaScript, this will be the only work necessary, so you should do it before you start other work and waste operations in the lead up to it.

Reply:
Ah, that makes sense. More straight forward approach. Thanks for the feedback!